api/v1alpha1/workspace_validation.go

```diff
@@ -86,7 +88,27 @@ func (w *Workspace) validateUpdate(old *Workspace) (errs *apis.FieldError) {
 	return errs
 }
 
-func (r *TuningSpec) validateCreate() (errs *apis.FieldError) {
+func (r *TuningSpec) validateCreate(ctx context.Context, workspaceNamespace string) (errs *apis.FieldError) {
+	methodLowerCase := strings.ToLower(string(r.Method))
+	if methodLowerCase != string(TuningMethodLora) && methodLowerCase != string(TuningMethodQLora) {
+		errs = errs.Also(apis.ErrInvalidValue(r.Method, "Method"))
+	}
+	if r.ConfigTemplate == "" {
```

Comment:
Here is the suggestion:

If the congfigTemplate is empty, we still validate the defaultLoraConfigMap and defaultQloraCongMap because users may change the contents.

As long as the configTemplate is specified, the configMap has to be in WS's namespace. This is consistent with the API definition.

Reply:
done